sqlalchemy-stubs/orm/decl_api.pyi

```diff
@@ -25,13 +34,10 @@ class DeclarativeMeta(type):
     def __setattr__(cls, key: Any, value: Any) -> None: ...
     def __delattr__(cls, key: Any) -> None: ...
     metadata: MetaData
-    registry: "registry"
+    registry: _registry  # Avoid circural reference
```

Comment:
how does this part work?  how is "_registry" different from "registry" ?

Reply:
"registry" refers to `DeclarativeMeta.registry` which is a circural reference.